### ISSUE TITLE
It seems that 2.0.0-M1 cannot be resolved from the repository

### ISSUE THREAD

**Issue by WeiZheng** [edulify/play-hikaricp.edulify.com#51]
Hi,

After I added 

```
resolvers += Resolver.url("Edulify Repository", url("http://edulify.github.io/modules/releases/"))(Resolver.ivyStylePatterns)
```

to the build.sbt, the module still cannot be resolved.

The scala version is set to 2.11.5:
```
scalaVersion := "2.11.5"
```

Am I wrong?

Best Regards,
Wei Zheng

**Comment by WeiZheng** [comment]
2.0.0-M1 cannot be resolved, 1.5.2 is fine.

**Comment by megazord** [comment]
Hello, @WeiZheng 

This was not released because it is a development version not intended to be used in production. 1.5.2 is the last stable version. I will probably release a final version this week, since most of the changes are pretty stable in my development/qa environments.
